test/test-mda-models/test-data-delivery-spark-model/pom.xml

```diff
@@ -248,5 +228,17 @@
             <version>${version.cucumber.reporting.plugin}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-client</artifactId>
+            <version>3.15.6.Final</version>
+            <scope>test</scope>
```

Comment:
I: Had to pin this version as investigated for few days and other versions doesn't seems to be compatible with this error 
.AbstractMethodError: Receiver class org.jboss.resteasy.microprofile.config.ServletConfigSource does not define or inherit an implementation of the resolved method 'abstract java.util.Set getPropertyNames()' of interface org.eclipse.microprofile.config.spi.ConfigSource.

Reply:
Q: Would this end up impacting downstream projects? If so, we will need to make sure we're tracking and resolving this issue before the release of 1.10, as I'm assuming this was caused by the Java 17 upgrade.

Reply:
can we add this version as a property in `build-parent/pom.xml` to keep things consistent

Reply:
I can create downstream project for spark to see if it is impacting!